model/test_results.go

```diff
@@ -446,6 +446,13 @@ type TestResultsStats struct {
 	FailedCount int `bson:"failed_count"`
 }
 
+// TestResultsSample contains test names culled from a test result's FailedTestsSample.
+type TestResultsSample struct {
```

Comment:
I remember the UI team also wanting the stats for these so that they can display `X/Y failing tests match` for each task id on the task history page. Do they still want that?

Reply:
Yes we do I forgot I mentioned this to Julian and not Jonathan. Sorry

Reply:
I updated it to also return the total length of the cache (and the aggregate length of all the execution task caches in the case of a display task).